core/src/main/java/io/questdb/log/LogFactory.java

```diff
@@ -975,11 +975,26 @@ public LogRecord microTime(long x) {
             return this;
         }
 
+        @Override
+        public LogRecord put(@Nullable Utf8Sequence us) {
+            return this;
+        }
+
+        @Override
+        public LogRecord put(byte b) {
+            return null;
+        }
+
         @Override
         public LogRecord put(char c) {
             return this;
         }
 
+        @Override
+        public LogRecord putUtf8(long lo, long hi) {
+            return null;
```

Comment:
Should return `this` to avoid NPE

Reply:
fixed in https://github.com/questdb/questdb/pull/4137/commits/4f372f37e5cf4d5b7f9e8374383a72b79d8eee10